FREY-1234: Review contribution guidelines (#382)
ScilifelabDataCentre/data.scilifelab.se: README.md

git pull upstream develop
```

> When you have finished editing, commit and push to your fork:

Reviewer: Just check - do we always recommend forking? Users often will create branches. we should be consistent

Author: I took most of it from the existing guidelines we have on Confluence, so I was wondering... The advantage I see it that it may be easier for non-techy persons to make contributions as this is similar process to copying and modifying something, and it avoids us the hassle of guiding them through the branch creation process, checkout, merging.

We can as well keep the fork strategy for external contributors only, and we keep working with branches within the team. But no strong opinion here I'll go with the general consensus :)

Author: @senthil10 just realised I still had work to do on this one... :sweat_smile: 
What do you think of Liane's comment though regarding what has been advised in the past?

Reviewer 2: Yes, external people can't create branches anyway I guess (due to repo settings). So yeah, I too agree external collaborators can fork and internal team member can crate new branches 👍

Author: Ok then I will let it like it is now, as CONTRIBUTING.md is mostly geared towards external contributors anyway

git pull upstream develop
```

> To keep commit messages consistent, contributors are advised to configure `git` with the supplied commit message template by running the following command in the repository's directory:

Reviewer: ```suggestion
To keep commit messages consistent, contributors are advised to configure `git` with the commit message template provided. This can be achieved by running the following command in the repository's directory:
```

```bash
git config --local commit.template .github/.gitmessage.txt
```

> This will cause `git` to pre-fill your commit message with the following template:

Reviewer: ```suggestion
The commit message will then be pre-filled with the following template:
```

```text
<type>(scope): <short description>
```

When you have finished editing, commit and push to your fork.

> For a good commit message, just replace the `<type>` and `(scope)` placeholders and write what would follow after "*If accepted, my changes are going to...*" as a short description:

Reviewer: ```suggestion
Replace the `<type>` and `(scope)` placeholders with (??). For the short description, write as if completing "*If accepted, my changes are going to...*":
```
```suggestion
For a good commit message, just replace the `<type>` and `(scope)` placeholders and write what would follow after "*If accepted, my changes are going to...*" as a short description:
```

Reviewer: I think that it would be good to provide examples for scope and type here.
